uber/templates/base.html

```diff
@@ -29,9 +29,19 @@
 
         <script src="../static/deps/combined.min.js" type="text/javascript"></script>
         <script src="../static/js/common-static.js" type="text/javascript"></script>
-        <script type="text/javascript">
-            var csrf_token = '{{ c.CSRF_TOKEN }}';
-        </script>
+
+        {% if not dont_show_csrf_token %}
+            {#
+                NOTE: never set dont_show_csrf_token to True unless you know exactly what you're doing.
+                this should only ever be used in situations where a page is cached for all users.
+                EXPERTS ONLY
+            #}
+            <script type="text/javascript">
+                var csrf_token = '{{ c.CSRF_TOKEN }}';
+            </script>
+        {% else %}
+            <!-- skipped showing CSRF token -->
+        {% endif %}
```

Comment:
I added the ability to leave the CSRF token off of the javascript page generation.  this is so when we're caching the schedule HTML, it doesn't have someone else's CSRF token in there.

I don't think this was hurting anything though technically it could have been a (farfetched) security risk for someone to know this.

I wanted the cached HTML from the scheduling page to be as static as possible.